remoteworkitem/trackerquery_repository_test.go

```diff
@@ -45,23 +45,32 @@ func (test *TestTrackerQueryRepository) TestTrackerQueryCreate() {
 	params := url.Values{}
 	ctx := goa.NewContext(context.Background(), nil, req, params)
 
-	query, err := test.queryRepo.Create(ctx, "abc", "xyz", uuid.NewV4(), space.SystemSpace)
-	assert.IsType(t, remoteworkitem.InternalError{}, err)
-	assert.Nil(t, query)
+	testFxt := tf.NewTestFixture(test.T(), test.DB, tf.Spaces(1))
+
+	tq := remoteworkitem.TrackerQuery{
+		Query:     "abc",
+		Schedule:  "xyz",
+		TrackerID: uuid.NewV4(),
+		SpaceID:   testFxt.Spaces[0].ID,
+	}
+
+	err := test.queryRepo.Create(ctx, &tq)
+	assert.IsType(t, errors.InternalError{}, err)
```

Comment:
Please organize this tests into sub tests. Otherwise I wouldn't know why you expect an error here.

Reply:
Btw. It is enough to call the function `TestCreate` instead of `TestTrackerQueryCreate`. The full context is visible through the suite name.

Reply:
Good. But please rename `testFxt` to `fxt` as we always do everywhere else.

Reply:
done here 35ce7c3